app.js

```diff
@@ -1,6 +1,12 @@
 const express = require("express");
 const app = express();
 const getAllTopics = require("./controllers/getAllTopics.controller")
+const endpoints = require("./endpoints.json")
+
+app.get("/api", (request, response) => {
+  response.status(200).send({endpoints})
```

Comment:
Nice use of shorthand for this object - be sure to keep it consistent as you move on through later tickets to make sure your code is as tidy as possible